apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -0,0 +1,739 @@
+          formatter: listFormatter,
+          switchable: true,
+        },
+        {
+          field: 'group_roles',
+          title: html` Roles
+            <button
+              class="btn btn-xs"
+              type="button"
+              title="Show roles help"
+              data-toggle="modal"
+              data-target="#role-help"
+            >
+              <i class="bi-question-circle-fill" aria-hidden="true"></i>
+            </button>`,
+          sortable: true,
+          class: 'text-center align-middle text-wrap',
+          formatter: uniqueListFormatter,
+          switchable: true,
+        },
+        ...nonspecificColumns,
+        actionButton,
+      ];
+    } else {
+      return [
+        assessmentInstanceIdColumn,
+        {
+          field: 'uid',
+          title: 'UID',
+          visible: false,
+          sortable: true,
+          class: 'align-middle text-nowrap',
+          switchable: true,
+        },
+        {
+          field: 'name',
+          title: 'Name',
+          sortable: true,
+          class: 'align-middle text-nowrap',
+          switchable: true,
+        },
+        {
+          field: 'role',
+          title: html` Role
+            <button
+              class="btn btn-xs"
+              type="button"
+              title="Show roles help"
+              data-toggle="modal"
+              data-target="#role-help"
+            >
+              <i class="bi-question-circle-fill" aria-hidden="true"></i>
+            </button>`,
+          sortable: true,
+          class: 'text-center align-middle text-nowrap',
+          switchable: true,
+        },
+        ...nonspecificColumns,
+        {
+          field: 'client_fingerprint_id_change_count',
+          title: html` Fingerprint Changes
+            <button
+              class="btn btn-xs"
+              ,
+              type="button"
+              title="Show figerprint changes help"
+              data-toggle="modal"
+              data-target="#fingerprint-changes-help"
+            >
+              <i class="bi-question-circle-fill" aria-hidden="true"></i>
+            </button>`,
+          class: 'text-center align-middle',
+          switchable: true,
+        },
+        actionButton,
+      ];
+    }
+  }
+
+  function refreshTable() {
+    bsTable.bootstrapTable('refresh', { silent: true });
+  }
+
+  window.popoverSubmitViaAjax = function (e: SubmitEvent, popover: JQuery) {
+    e.preventDefault();
+    if (e.target) {
+      $.post(
+        window.location.pathname,
+        $(e.target).serialize(),
+        function () {
+          refreshTable();
+        },
+        'json',
+      );
+      $(popover).popover('hide');
+    }
+  };
+
+  function timeLimitEditPopoverContent(this: any) {
+    const row = $(this).data('row');
+    const action = row.action ? row.action : 'set_time_limit';
+    const checkbox = $('#reopen-closed');
+    checkbox.toggle(row.time_remaining_sec === null);
```

Comment:
What's this doing? AFAICT the conditional display of this checkbox is handled around line 444.

Reply:
Good catch, removed these lines in [0b3aa97](https://github.com/PrairieLearn/PrairieLearn/pull/9725/commits/0b3aa97bfea69bfe855c4af0ac526d99182d6f6b).